### ISSUE TITLE
New CVEs reported in fluentd image

### ISSUE THREAD

**Issue by KR411-prog** [fluent/fluentd-kubernetes-daemonset#1281]
Fluentd image version: fluent/fluentd-kubernetes-daemonset:v1.13.2-debian-azureblob-1.0

<img width="1295" alt="image" src="https://user-images.githubusercontent.com/14121261/126566454-cca8e6c9-8352-4c0f-9531-ad6ff4a20a58.png">

**Comment by kenhys** [comment]
CVE-2021-30535

https://security-tracker.debian.org/tracker/CVE-2021-30535
```
# dpkg -l |\grep icu
ii  icu-devtools            63.1-6+deb10u1             amd64        Development utilities for International Components for Unicode
ii  libicu-dev:amd64        63.1-6+deb10u1             amd64        Development files for International Components for Unicode
ii  libicu63:amd64          63.1-6+deb10u1             amd64        International Components for Unicode
```

The affected version is installed, but it is caused via crafted HTML page in Google Chrome, so it is a false positive.

**Comment by kenhys** [comment]
CVE-2020-36327

https://bugzilla.redhat.com/show_bug.cgi?id=1958999
https://bundler.io/blog/2021/02/15/a-more-secure-bundler-we-fixed-our-source-priorities.html
at least, 2.2.18 is needed.

Fix dependency confusion issues with implicit dependencies
https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#security-fixes-1

In deamonset image side, gem was already installed explicitly, so even though vulnerable bundler is installed, it will not be affected if you use provided image as is.
In contrast to using provided image as is, If you make derivated  image, it may be affected when installing private gems.

**Comment by kenhys** [comment]
Fixed by #1284

**Comment by kenhys** [comment]
It should be fixed fluent/fluentd-kubernetes-daemonset:v1.13.3-debian-azureblob-1.1
